alacritty/src/renderer/rects.rs

```diff
@@ -193,6 +193,9 @@ impl RenderLines {
             return;
         }
 
+        // Strikethrough isn't using the same color underline is using.
+        let color = if flag.contains(Flags::STRIKEOUT) { cell.fg } else { cell.underline };
```

Comment:
Rather than stating it isn't, I'd also state why it's not:

```suggestion
        // The underline color escape does not apply to strikeout.
        let color = if flag.contains(Flags::STRIKEOUT) { cell.fg } else { cell.underline };
```